imx-core-sdk-kotlin-jvm/build.gradle

```diff
@@ -51,9 +51,6 @@ openApiGenerate {
             // AssetOrderDetails model (with List<SellOrders>) for 'Asset.orders' field to use.
             OrderDetails: "com.immutable.sdk.api.model.AssetOrderDetails"
     ]
-    additionalProperties = [
-            nonPublicApi: true
-    ]
```

Comment:
Can you also delete the `data_class.mustache` file? It only exists to remove the use of `nonPublicApi`.

Reply:
Removed `data_class.mustache`.

> Do we recommend that people use `ImmutableX.collectionsApi.listCollections(...)` or `CollectionsApi().listCollections(...)`? If we recommend the first we should also update the readme.

I will update the README once it is released, otherwise it won't reflect what is publicly available right now.